Doc/library/urllib.parse.rst

```diff
@@ -22,19 +22,11 @@ to an absolute URL given a "base URL."
 
 The module has been designed to match the internet RFC on Relative Uniform
 Resource Locators. It supports the following URL schemes: ``file``, ``ftp``,
-``gopher``, ``hdl``, ``http``, ``https``, ``imap``, ``itms-services``, ``mailto``, ``mms``,
+``gopher``, ``hdl``, ``http``, ``https``, ``imap``, ``mailto``, ``mms``,
```

Comment:
It's worth noting that while this is what would be a strict reverse of the original PR, this change was actually correct - the list didn't previously list `itms-services` at all, despite being supported.